src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs

```diff
@@ -97,5 +97,13 @@ public void TestExpressRouteCircuitPeeringWithRouteFilter()
         {
             TestRunner.RunTestScript("Test-ExpressRouteCircuitPeeringWithRouteFilter");
         }
-    }
+
+	    [Fact(Skip = "Rerecord tests")]
```

Comment:
So, you added a new test that is... skipped? Why not record it?

Reply:
I hit a few issues with the recording. I am working on it. Additionally, the feature is not GA-ed. Spoke with @MikhailTryakhov earlier on this.

Reply:
Nono, we need to enable it on test subscription and add the recording.

Overwise - you can request private preview build from powershell

Reply:
Thanks for clarifying. I will fix up the recording. Please go ahead with the branch cut; no need to wait for my merge.

Reply:
Thanks for flexibility @raraven. You can retarget to master and be a part of build release or wait for a next monthly branch. Currently closing this pr